dist/elm/ES/UI/Color.elm

```diff
@@ -25,76 +25,219 @@ transparent =
 
 
 teal =
-    { dark = Color.rgb255 16 128 139 -- #10808b
-    , base = Color.rgb255 20 161 174 -- #14a1ae
-    , light = Color.rgb255 0 195 201 -- #00c3c9
-    , lighter = Color.rgb255 102 219 223 -- #66dbdf
-    , lightest = Color.rgb255 218 245 247 -- #daf5f7
+    { dark = Color.rgb 0.06 0.5 0.55 -- #10808B
+    , base = Color.rgb 0.08 0.63 0.68 -- #14A1AE
+    , light = Color.rgb 0.0 0.76 0.79 -- #00C3C9
+    , lighter = Color.rgb 0.4 0.86 0.87 -- #66DBDF
+    , lightest = Color.rgb 0.85 0.96 0.97 -- #DAF5F7
```

Comment:
I needed to switch these because Elm uses 0-1 internally and I can't match on base 255.